modules/nw-sriov-example-dpdk-line-rate.adoc

```diff
@@ -0,0 +1,278 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/using-dpdk-and-rdma.adoc
+
+:_content-type: PROCEDURE
+[id="nw-example-dpdk-line-rate_{context}"]
+= Using SR-IOV and the Node Tuning Operator to achieve a DPDK line rate
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Install the SR-IOV Network Operator.
+* Log in as a user with `cluster-admin` privileges.
+* Deploy a standalone Node Tuning Operator.
+
+The Node Tuning Operator is used to configure isolated CPUs, hugepages, and a topology scheduler.
+You can then use the Node Tuning Operator with SR-IOV to achieve a specific DPDK line rate networking.
+
+[NOTE]
+====
+In previous versions of OpenShift Container Platform, the Performance Addon Operator was used to implement automatic tuning to achieve low latency performance for OpenShift applications. In OpenShift Container Platform 4.11, this functionality is part of the Node Tuning Operator.
+====
+
+.Procedure
+
+== Using the Node Tuning Operator to optimize line rate
+
+.Procedure
+. Create a `PerformanceProfile` object based on the following example:
++
+[source,yaml]
+----
+    apiVersion: performance.openshift.io/v2
+    kind: PerformanceProfile
+    metadata:
+     name: performance
+    spec:
+     globallyDisableIrqLoadBalancing: true
+     cpu:
+       isolated: 21-51,73-103 <1>
+       reserved: 0-20,52-72 <2>
+     hugepages:
+       defaultHugepagesSize: 1G <3>
+       pages:
+       - count: 32
+         size: 1G
+     net:
+        userLevelNetworking: true
+     numa:
+       topologyPolicy: "single-numa-node"
+     nodeSelector:
+       node-role.kubernetes.io/worker-cnf: ""
+----
+<1> If hyperthreading is enabled on the system, allocate the symbolic link to the same CPU group (isolated/reserved). If the system contains multiple NUMAs, allocate CPUs from both NUMAs to both groups. You can also use the Performance Profile Creator for this task: see "Creating a performance profile" for more information.
+<2> You can also specify a list of devices that will have the queues set to the reserved CPU count. See "Reducing NIC queues using the Node Tuning Operator" for more information.
+<3> Allocate the number and size of hugepages needed. You can specify the NUMA for the hugepages. By default, the system allocates an even number to every NUMA on the system. If needed, you can request the use of a realtime kernel for the nodes. See "Provisioning a worker with real-time capabilities" for more information.
+. Save the `yaml` file as `mlx-dpdk-perfprofile-policy.yaml`.
+. Apply the performance profile.
+
+== Configuring an SR-IOV Network Operator
+The SR-IOV network operator is used to allocate and configure Virtual Functions from SR-IOV-supporting Physical Function NICs on the nodes.
+
+For more information on deploying the operator, see "Installing the SR-IOV Network Operator".
```

Comment:
You could change `"Installing the SR-IOV Network Operator"` to `_Installing the SR-IOV Network Operator_`.